int8 sessionLength
string answers
---
int8 resultCode